src/gui/MainWindow.cpp

```diff
@@ -552,13 +555,20 @@ void MainWindow::finalize()
 								m_toolBar );
 	controllers_window->setShortcut( Qt::Key_F11 );
 
+    m_countDownClock = new LcdSpinBox(2, this, tr("Count Down Clock"));
+
+    m_countDownClock->setLabel( tr("SEC") );
```

Comment:
The countdown should really be in bars/beats, not seconds. When you record you always want to be relative to the beat, because the countdown actually gives musicians time to align themselves with the beat. An arbitrary second value can't do that.